bash_scripting_101/bash_scripting_101.md

```diff
@@ -37,241 +39,443 @@ script: https://kit.fontawesome.com/83b2343bd4.js
 
-## Useful Commands and Syntax
+- Move to a directory relative to your present working directory:
 
-In our installation section, we covered how accessing CLI tools to run bash scripts is quite uncomplicated on either operating system. Similarly, bash commands and syntax are approachable and integrate well to allow for a gradual increasing in the complexity of utilities run.
+  * If the directory `directory_name` is in your present working directory (i.e. it is one of the listed items when you enter `ls`) you can navigate into it with the command `cd directory_name`
 
-### Commands
+  * To move to the parent directory of your current location, use `cd ..`
 
-- `Ls`: list directory contents
-- `Mkdir`: create a new directory at the folder level
-- `Cd`: change directory
-- `Echo`: prints out text in the terminal window- especially useful for declaring environment variables which reveal both permissions and what strings can be passed.
+  * Use slashes `/` to link directions. For example `cd ../..` up to the the parent directory of your parent directory.
 
-  ![Echo variable example demonstrating how to print and set the price of a pint.](media/echo_example_large.png)
+  * It is possible to move up and then back down in your file hierarchy. For example `cd ../other_directory` would move you to the `other_directory` that shares the same parent directory as your current location.
 
-- `Touch`: creates a new file
+- Move to a directory using its file path:
 
-  - `touch file 1` will create a single new file
-  - `touch file 1 file 2 file 3` will create multiple uniquely named files at once
+  * To return to your home directory use `cd ~`
 
-- `Grep`: powerful search function to locate a string or pattern within a file or in a directory. Grep stands for global regular expression print.
+  * If you know how to get to the directory you want from your home directory, you can go directly there with `cd ~/fixed/path/to/directory`
 
-  - `grep 'panda' file 1 file 2 file 3` will search for the word "panda" in the three files we just created.
 
-- `Man`: reveals function of a given command
+**Give it a try!** Use these commands to explore the directories on your computer. Try navigating around some files and folders that you are already familiar with to get comfortable with how they look through the bash shell.
 
-  ![Grep man command output with the full definition and some options displayed.](media/grep_output_large.png)
+### Tab completion
 
-- `Mv`: move or rename a directory and allows for batch processing
-  `mv *.txt folder` will move all files with a .txt file extension into a folder for example.
-- `Less`: view contents of a file without opening a separate editor
+If it seems like there is an awful lot to make sure you are typing correctly  into your bash shell, you are right. Luckily you don't have to type everything out fully because you can use the `tab` key on your computer.
```

Comment:
```suggestion
If it seems like there is an awful lot to make sure you are typing correctly into your bash shell, you are right. Luckily you don't have to type everything out fully because you can use the `tab` key on your computer.
```